e2e/cypress/integration/scroll/default_images_collapsed_spec.js

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Scroll', () => {
+    let testTeam;
+
+    before(() => {
+        // # Create new team and new user and visit Town Square channel
+        cy.apiInitSetup({loginAfter: true}).then(({team}) => {
+            // # Switch the account settings for the test user to have images collapsed by default
+            cy.apiSaveCollapsePreviewsPreference('true');
+
+            testTeam = team;
+            cy.visit(`/${testTeam.name}/channels/town-square`);
```

Comment:
I think no need to declare `testTeam` since it's only used here where `team` can be used directly.
```suggestion
            cy.visit(`/${team.name}/channels/town-square`);
```

Reply:
Though not mentioned, the test assumed several posts before the collapsed image. I'll update the test case but could you please add below after visiting town-square channel,
```javascript
Cypress._.times(10, (index) => cy.postMessage(index));
```